lib/bemhtml/index.js

```diff
@@ -180,7 +180,16 @@ BEMHTML.prototype.renderClose = function renderClose(prefix,
     if (content || content === 0)
       out += this.renderContent(content, isBEM);
 
-    out += '</' + tag + '>';
+    var optionalClosingTags = [
```

Comment:
@vithar if you will be [there](https://github.com/bem/bem-xjst/blob/master/lib/bemxjst/utils.js#L43-L46) change comment text to english please.